apiserver/facades/agent/machineactions/machineactions_test.go

```diff
@@ -179,6 +175,75 @@ func (s *facadeSuite) TestActionsTaskNotPending(c *tc.C) {
 	c.Check(results.Results[0].Error, tc.ErrorMatches, "action no longer available")
 }
 
+func (s *facadeSuite) TestRunningActionsReturnsActionIDsForMachine(c *tc.C) {
+	defer s.setupMocks(c).Finish()
+
+	// Arrange
+	machineTag := names.NewMachineTag(s.authTag.Id())
+	s.operationService.EXPECT().GetMachineTaskIDsWithStatus(
+		gomock.Any(), coremachine.Name(s.authTag.Id()), status.Running,
+	).Return([]string{"42", "47"}, nil)
+
+	// Act
+	results := s.getFacade().RunningActions(c.Context(), params.Entities{Entities: []params.Entity{{Tag: machineTag.String()}}})
+
+	// Assert
+	c.Assert(results.Actions, tc.HasLen, 1)
+	c.Check(results.Actions[0].Receiver, tc.Equals, machineTag.String())
+	c.Check(results.Actions[0].Error, tc.IsNil)
+	c.Check(results.Actions[0].Actions, tc.HasLen, 2)
+	c.Check(results.Actions[0].Actions[0].Action.Tag, tc.Equals, names.NewActionTag("42").String())
+	c.Check(results.Actions[0].Actions[1].Action.Tag, tc.Equals, names.NewActionTag("47").String())
+}
+
+func (s *facadeSuite) TestRunningActionsReturnsActionIDsForMachineEmptyArgs(c *tc.C) {
+	defer s.setupMocks(c).Finish()
+
+	// Arrange
+	s.operationService.EXPECT().GetMachineTaskIDsWithStatus(
+		gomock.Any(), coremachine.Name(s.authTag.Id()), status.Running,
+	).Times(0) // not called
```

Comment:
delete me: this isn't needed, the test will already fail if `GetMachineTaskIDsWithStatus` is called.